Resources/Locale/en-US/components/space-heater-component.ftl

```diff
@@ -0,0 +1,13 @@
+﻿comp-space-heater-ui-thermostat = Thermostat :
+comp-space-heater-ui-toggle = Toggle
```

Comment:
I don't personally think a label for this is necessary, but it's fine if you want to keep it.